### ISSUE TITLE
ACT Mapping: map ACT rules against Checker

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#982]
- [x] Map ACT rules against Checker
- [x] Identify coverage
- [x] Identify disagreements (inconsistencies)
- [x] Take proposal to Subject Matter Connection (SME Connection) for review
- [x] Open new issue to fix Checker rule reporting - see 
- #1016 
- #1018 
- [x] Open new issue to fix Help references #1019
- [x] Open new issues for Checker inconsistencies for Fail example that Checker passes (or ignores) 
- #1020 
- #1021
- #1022
- [x] Open new issue for recommendations for mapping changes top ACT community [#4763](https://github.ibm.com/IBMa/E2E/issues/4763)

**Comment by philljenkins** [comment]
**_Examples_** of inconsistency between ACT test case expected results and Checker results 

<html>
<body>
<google-sheets-html-origin>

ACT Rule | ACT Test Case | Expected | Checker Results | RuleID
-- | -- | -- | -- | --
[ARIA required context role](https://act-rules.github.io/rules/ff89c9) | [Failed Example 3](https://act-rules.github.io/rules/ff89c9/#failed-example-3) | Failed | Passed | [Rpt_Aria_RequiredParent_Native_Host_Sematics](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/Rpt_Aria_RequiredParent_Native_Host_Sematics.ts)
[ARIA required owned elements](https://act-rules.github.io/rules/bc4a75) | [Failed Example 3](https://act-rules.github.io/rules/bc4a75/#failed-example-3) | Failed | Passed | [Rpt_Aria_RequiredChildren_Native_Host_Sematics]()


</body>
</html>

**Comment by philljenkins** [comment]
Was blocked by #984

**Comment by marcjohlic** [comment]
Phill to check update from Tom (url) to see if that unblocks

**Comment by tombrunet** [comment]
For comparison, AXE results: https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/axe-core/

**Comment by philljenkins** [comment]
@tombrunet 
Completed the ACT/Checker mapping for which Requirement is being reported against:
See [Checker_ACT_Mapping 2022-06-23.xlsx](https://ibm.ent.box.com/file/981548094874)
(see column H cells highlighted in yellow for which Checker rules need changing)
(see column I cells highlighted in orange for which ACT rules need mapping changes)

In summary, the following six Checker rules need mapping changes for which Requirement is being reported:

1. [WCAG20_Input_ExplicitLabel.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/WCAG20_Input_ExplicitLabel.ts)
1. [WCAG20_Elem_Lang_Valid.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/WCAG20_Elem_Lang_Valid.ts)
1. [HAAC_Aria_ImgAlt.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/HAAC_Aria_ImgAlt.ts)
1. [HAAC_Aria_SvgAlt.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/HAAC_Aria_SvgAlt.ts)
1. [WCAG20_Frame_HasTitle.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/WCAG20_Frame_HasTitle.ts)
1. [meta_viewport_zoom.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/meta_viewport_zoom.ts)

![image](https://user-images.githubusercontent.com/16193609/177926336-ec98a315-1789-4b20-97cf-1dbbd9327054.png)

**Comment by MHoov** [comment]
@philljenkins this issue hasn't been updated in 21 days.
@marcjohlic FYI

**Comment by philljenkins** [comment]
Analysis reported to SME council July 21.
Opening new issues as document in issue description above.

**Comment by philljenkins** [comment]
Will add related issues here in comments if they do not show up in the description above.

**Comment by philljenkins** [comment]
Opened issues for follow-on work to improve mapping and failure examples
